pkg/primitives/statistics.go

```diff
@@ -175,6 +187,8 @@ func NewStatisticsMessageBus(router rmb.Router, stats *Statistics) error {
 func (s *statisticsMessageBus) setup(router rmb.Router) error {
 	sub := router.Subroute("statistics")
 	sub.WithHandler("get", s.getCounters)
+	sub.WithHandler("deployments_count", s.deploymentsCount)
+	sub.WithHandler("workloads_count", s.workloadsCount)
```

Comment:
This means that the user of those counters need to make 2 calls to get the node statistics! it's a lot of round trips for related data. 

Instead please extend the getCounters function to return all node statistics. it already return `total`, `used`, and `system` can also have a sub object 
```json
"total": ...,
"used": ...,
"system": ...,
"user": {
    "deployments": <number>,
    "workloads": <number>,
}
```

this way the caller can get all node statistics in a single call